### PR TITLE
Abstract

### PR DESCRIPTION
**Pull request by dmurvihill**
Here is an abstract!

### CODE REVIEW

File: doc/src/abstract.ltx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+In response to growing academic dishonesty in low-level computer science and
```
Comment (mheon) [inline doc/src/abstract.ltx]: Recommend adding evidence in here --- "growing evidence of academic dishonesty"

Comment (dmurvihill) [inline doc/src/abstract.ltx]: Why? Is the evidence not strong enough to conclude that academic dishonesty is growing?

Comment (mheon) [inline doc/src/abstract.ltx]: I dunno if we want to make such a statement, given we never charted it vs time?

Comment (dmurvihill) [inline doc/src/abstract.ltx]: But we did chart the evidence vs time?

Comment (mheon) [inline doc/src/abstract.ltx]: Fair point... I still think Lauer might take issue, but we can cross that when we come to it.

**Pull request merged**